src/app/shell/personal-cabinet/provider/create-provider-admin/create-provider-admin.component.ts

```diff
@@ -169,16 +174,22 @@ export class CreateProviderAdminComponent extends CreateFormComponent implements
   }
 
   onSubmit(): void {
+    let confirmationType: string;
+
+    if (this.editMode) {
+      this.isDebuty ? 
```

Comment:
You can write it in another way but I guess both ways should work fine
`confirmationType = this.isDebuty ? ModalConfirmationType.updateProviderAdminDeputy : ModalConfirmationType.updateProviderAdmin;`